internal/declcfg/declcfg_to_model.go

```diff
@@ -66,6 +68,23 @@ func ConvertToModel(cfg DeclarativeConfig) (model.Model, error) {
 				}
 				mpkg.Channels[bundleChannel.Name] = pkgChannel
 			}
+
+			// Parse version from the package property, falling back to the CSV's spec.version field.
```

Comment:
@joelanford opinions on this being a bundle field? Pre-parsing version makes comparison way easier and more efficient.

Reply:
Apologies, I meant on the `model.Bundle` type itself. I agree with what you said above.

I could also make the `olm.package`'s Go `Version` field a `semver.Version`.

Reply:
`Version` on `model.Bundle` seems reasonable, I suppose. I think we would need to add a bundle validation for `model.Bundle.Version == model.Bundle.Properties["olm.package"].Value.Version`

And then always make sure to populate that field when converting everything to the model.

Reply:
Mind if I handle this in a follow-up?